modules/nf-core/celesta/meta.yml

```diff
@@ -0,0 +1,67 @@
+---
+name: "celesta"
+description: Unsupervised machine learning for cell type identification in multiplexed imaging using protein expression and cell neighborhood information without ground truth
+keywords:
+  - highly_multiplexed_imaging
+  - cell_type_identification
+  - cell_phenotyping
+  - image_analysis
+  - mcmicro
+  - machine_learning
+tools:
+  - "celesta":
+      description: "Automate unsupervised machine learning cell type identification using both protein expressions and cell spatial neighborhood information"
+      homepage: "https://github.com/plevritis-lab/CELESTA"
+      documentation: "https://github.com/plevritis-lab/CELESTA/blob/main/README.md"
```

Comment:
add your repo with the Celesta CLI you've implemented for documentation instead here